src/test/java/com/wikia/webdriver/testcases/adstests/TestAmazonAds.java

```diff
@@ -61,10 +35,10 @@ private void testAmazonAd(String wikiName, String path, boolean debugMode) {
     if (debugMode) {
       testedPage = urlBuilder.appendQueryStringToURL(testedPage, "amzn_debug_mode=1");
     }
-    AdsAmazonObject amazonAds = new AdsAmazonObject(driver, testedPage, networkTrafficInterceptor);
+    AdsAmazonObject amazonAds = new AdsAmazonObject(driver, testedPage);
 
     amazonAds.verifyAmazonScriptIncluded();
-    amazonAds.verifyCallToAmazonIssued();
+//    amazonAds.verifyCallToAmazonIssued();
```

Comment:
I would like to keep the commented line since it reminds me about the problem with `browsermob-proxy` and I am going to uncomment the line when the issue will be fixed.